docs/src/components/navigation/social-links.tsx

```diff
@@ -0,0 +1,83 @@
+import Link from 'next/link';
+
+export function SocialLinks({ tabIndex }: { tabIndex?: number }) {
+  const styles =
+    'shrink-0 rounded-lg p-3 hover:bg-keystatic-gray-light active:bg-keystatic-gray transition-colors';
+
+  return (
+    <div className="flex gap-2">
+      <Link
+        href="https://fosstodon.org/@keystatic"
+        className={styles}
+        tabIndex={tabIndex}
+      >
+        <span className="sr-only">Keystatic on Mastodon</span>
+        <svg
+          className="h-4 w-4"
+          xmlns="http://www.w3.org/2000/svg"
+          fill="none"
+          viewBox="0 0 16 16"
+        >
+          <g clipPath="url(#a)">
+            <path
+              fill="#000"
+              fillRule="evenodd"
+              d="M11.941.206c1.776.262 3.275 1.617 3.506 3.336.1 1.023.05 2.56.027 3.306l-.011.39c0 .101-.015 1.017-.02 1.114-.156 2.457-1.688 3.427-3.297 3.736l-.058.013-.015.003c-1.02.199-2.114.252-3.151.281-.248.007-.495.007-.743.007a13.04 13.04 0 0 1-3.062-.364.035.035 0 0 0-.044.036c.029.326.1.647.212.955.139.357.624 1.213 2.427 1.213 1.049.002 2.093-.12 3.113-.364a.035.035 0 0 1 .016 0c.005.001.01.003.014.007a.036.036 0 0 1 .013.027v1.206a.038.038 0 0 1-.015.029c-.319.23-.747.367-1.123.486l-.05.016c-.17.054-.344.1-.519.142a8.792 8.792 0 0 1-4.796-.254c-1.442-.506-2.914-1.746-3.278-3.237a17.596 17.596 0 0 1-.41-2.454C.596 8.932.568 8.027.54 7.12.53 6.78.52 6.437.506 6.095c-.034-.87-.015-1.82.17-2.677C1.057 1.678 2.634.46 4.361.206a4.68 4.68 0 0 0 .12-.02C4.824.126 5.537 0 7.856 0h.02c2.63 0 3.766.161 4.065.206Zm.956 9.539V5.474c0-.873-.222-1.567-.667-2.08-.461-.513-1.064-.776-1.812-.776-.865 0-1.52.333-1.956.998l-.421.707-.422-.707c-.436-.665-1.09-.998-1.957-.998-.748 0-1.351.263-1.811.776-.446.514-.668 1.207-.668 2.08v4.27h1.691V5.6c0-.873.368-1.318 1.103-1.318.813 0 1.221.526 1.221 1.566v2.27h1.683v-2.27c0-1.04.407-1.566 1.22-1.566.74 0 1.103.445 1.103 1.318v4.145h1.693Z"
+              clipRule="evenodd"
+            />
+          </g>
+          <defs>
+            <clipPath id="a">
```

Comment:
Is that `id` correct? Line 75 has the same id. I'm thinking these need a unique id from `useId`?

Reply:
Ahh good catch, this is all copypaste from old, so haven't even noticed that, but yes will sort 🙌🏻